node/network/bridge/src/tx/tests.rs

```diff
@@ -20,9 +20,10 @@ use polkadot_node_subsystem_util::TimeoutExt;
 
 use async_trait::async_trait;
 use parking_lot::Mutex;
-use std::{borrow::Cow, collections::HashSet};
+use std::collections::HashSet;
 
 use sc_network::{Event as NetworkEvent, IfDisconnected};
+use sc_network_common::protocol::ProtocolName;
```

Comment:
Is it possible to re-export in `sc_network` so we don't have to import a separate crate for this?

Reply:
Done